src/plugins/fs/mount.c

```diff
@@ -689,19 +691,19 @@ gboolean bd_fs_mount (const gchar *device, const gchar *mountpoint, const gchar
     if (extra) {
         for (extra_p=extra; *extra_p; extra_p++) {
             if ((*extra_p)->opt && (g_strcmp0 ((*extra_p)->opt, "run_as_uid") == 0)) {
-                run_as_uid = g_ascii_strtoull ((*extra_p)->val, NULL, 0);
+                run_as_uid = g_ascii_strtoull ((*extra_p)->val, endptr, 0);
```

Comment:
Should be `&endptr` with `gchar *endptr`, AFAICT.

Reply:
Oops, fixed.